libs/feature/catalog/src/lib/my-org/my-org.service.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { TestBed } from '@angular/core/testing'
+import { MyOrgService } from './my-org.service'
+import { AuthService } from '@geonetwork-ui/api/repository/gn4'
+import { OrganizationsServiceInterface } from '@geonetwork-ui/common/domain/organizations.service.interface'
+import { BehaviorSubject, of, Subject } from 'rxjs'
+import { UserApiModel } from '@geonetwork-ui/data-access/gn4'
+import { UserModel } from '@geonetwork-ui/common/domain/user.model'
+import { HttpClientTestingModule } from '@angular/common/http/testing'
+import { TranslateService } from '@ngx-translate/core'
+import { AvatarServiceInterface } from '@geonetwork-ui/api/repository/gn4'
+
+const translateServiceMock = {
+  currentLang: 'fr',
+}
+
+class AvatarServiceInterfaceMock {
+  placeholder = 'http://placeholder.com'
+  getProfileIcon = (hash: string) => `${hash}`
+}
+
+describe('MyOrgService', () => {
+  let myOrgService: MyOrgService
+  let authService: AuthService
+  let orgService: OrganizationsServiceInterface
+
+  const orgs = [
+    { name: 'Géo2France', logoUrl: { href: 'logo-url' }, recordCount: 10 },
+  ]
+  const orgs$ = of(orgs)
+
+  const orgServiceMock = {
+    organisations$: orgs$,
+  }
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        MyOrgService,
+        {
+          provide: TranslateService,
+          useValue: translateServiceMock,
+        },
+        {
+          provide: AvatarServiceInterface,
+          useClass: AvatarServiceInterfaceMock,
+        },
+        { provide: OrganizationsServiceInterface, useValue: orgServiceMock },
```

Comment:
It would be nice if you could make `orgServiceMock` a class instead of a value, as it would be more consistent with other places where we inject mock services

Reply:
Done!